autopush/router/apns2.py

```diff
@@ -101,13 +101,14 @@ def send(self, router_token, payload, apns_id,
         """
         body = json.dumps(payload)
         priority = APNS_PRIORITY_IMMEDIATE if priority else APNS_PRIORITY_LOW
+        # NOTE: Hyper requires that all header values be strings.
         headers = {
             'apns-id': apns_id,
-            'apns-priority': priority,
+            'apns-priority': str(priority),
```

Comment:
I knew that this might get pinged, but would rather leave it in place. The problem is that the original priority values are integers, that are converted to const strings. There in lay dragons. converting a string to a string is a noop, but prevents any future issues should those values "revert" to integers. That's also why I added the comment.